tests/functional/ui/test_error_improvements.py

```diff
@@ -139,7 +140,8 @@ def test_blocking_pool_creation_rules(self, cephblockpool_factory_ui_class):
         blocking_pool_tab.check_edit_labels(block_pool_obj.name)
 
         blocking_pool_tab.proceed_resource_creation()
-        blocking_pool_tab.do_click(blocking_pool_tab.bp_loc["pool_type_block"])
+        if blocking_pool_tab.ocs_version_semantic >= version.VERSION_4_17:
```

Comment:
can you please override method 
`proceed_resource_creation` at the BlockPools class and insert this logic there?
It will look like 
```
def proceed_resource_creation(self):
  super().proceed_resource_creation()
  if blocking_pool_tab.ocs_version_semantic >= version.VERSION_4_17:
    self.do_click(self.bp_loc["pool_type_block"])
```

Reply:
Done